filter/diff_filter.go

```diff
@@ -138,6 +138,19 @@ func (df *DiffFilter) ShouldReport(path string, lnum int) (bool, *diff.FileDiff,
 	return df.isSignificantLine(line), file, line
 }
 
+func (df *DiffFilter) DiffLine(path string, lnum int) *diff.Line {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method DiffFilter.DiffLine should have comment or be unexported